File: modulehome/src/main/java/com/zht/modulehome/activity/ComposeActivity.java
package com.zht.modulehome.activity;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.alibaba.android.arouter.facade.annotation.Route;
import com.zht.common.base.BaseViewBindingActivity;
import com.zht.common.constant.ARoutePathConstants;
import com.zht.modulehome.R;
import com.zht.modulehome.databinding.ActivityJetpackBinding;

/**
 * @Date 2023/1/12 14:35
 * @Author zhanghaitao
 * @Description
 */
@Route(path = ARoutePathConstants.Home.COMPOSE_ACTIVITY)
public class ComposeActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_compose);
    }
}

File: modulehome/src/main/java/com/zht/modulehome/activity/ComposeActivity.kt
package com.zht.modulehome.activity

import android.os.Bundle
import com.alibaba.android.arouter.facade.annotation.Route
import com.zht.common.base.BaseViewBindingActivity
import com.zht.common.constant.ARoutePathConstants
import com.zht.modulehome.R
import com.zht.modulehome.databinding.ActivityComposeBinding

/**
 * @Date 2023/1/17 10:29
 * @Author zhanghaitao
 * @Description
 */
@Route(path = ARoutePathConstants.Home.COMPOSE_ACTIVITY)
class ComposeActivity : BaseViewBindingActivity<ActivityComposeBinding>() {

    override fun initView(savedInstanceState: Bundle?) {
        supportFragmentManager?.apply {
            this.beginTransaction()?.apply {
                add(R.id.compose_container, ComposeFragment(), "ComposeFragment")
                commitAllowingStateLoss()
                executePendingTransactions()
            }
        }
    }

}